cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -166,6 +166,11 @@
     <!-- End for JMS Source -->
     <!-- Start for Kafka Source -->
     <dependency>
+      <groupId>org.apache.zookeeper</groupId>
+      <artifactId>zookeeper</artifactId>
+      <scope>compile</scope>
```

Comment:
Except for the scope, the other change should be in release branch. During conflict if any, make sure the compile scope option is included